config/crd/bases/submariner.io_submariners.yaml

```diff
@@ -314,6 +335,33 @@ spec:
                             - public_ip
                             - subnets
                             type: object
+                          latency:
+                            description: LatencySpec describes the round trip time
+                              information in nanoseconds for a packet between the
```

Comment:
It's no longer nanoseconds.

Reply:
This is for the old fields it will be removed later. This is an auto-generated yaml.